docs/newbs_getting_started.md

```diff
@@ -1,99 +1,97 @@
 
-Once you have setup your Linux/Unix environment you are ready to download QMK. We will do this by using git to "clone" the QMK repository. Open a Terminal or MSYS2 Console window and leave it open for the remainder of this guide. Inside that window run these two commands:
+## Set Up QMK
+
+Once you have set up your Linux/Unix environment, you are ready to download QMK. We will do this by using Git to "clone" the QMK repository. Open a Terminal or MSYS2 Console window and leave it open for the remainder of this guide. Inside that window run these two commands:
 
     git clone https://github.com/qmk/qmk_firmware.git
     cd qmk_firmware
 
-?> If you already know [how to use GitHub](getting_started_github.md) we recommend you create and clone your own fork instead. If you don't know what that means you can safely ignore this message.
-
-## Setup QMK
+?> If you already know [how to use GitHub](getting_started_github.md), we recommend that you create and clone your own fork instead. If you don't know what that means, you can safely ignore this message.
 
-QMK comes with a script to help you setup the rest of what you'll need. You should run it now by typing in this command:
+QMK comes with a script to help you set up the rest of what you'll need. You should run it now by typing in this command:
 
-    ./util/qmk_install.sh
+    util/qmk_install.sh
 
 ## Test Your Build Environment
 
-Now that your QMK build environment is setup you can build a firmware for your keyboard. Start by trying to build the default layout for your keyboard. You should be able to do that with a command in this format:
+Now that your QMK build environment is set up, you can build a firmware for your keyboard. Start by trying to build the keyboard's default layout. You should be able to do that with a command in this format:
 
     make <keyboard>:default
 
-For example, to build a firmware for a Clueboard 66% use:
+For example, to build a firmware for a Clueboard 66% you would use:
 
     make clueboard/66/rev3:default
 
 When it is done you should have a lot of output that ends similar to this:
 
 ```
-Linking: .build/clueboard_66_rev2_default.elf                                                       [OK]
-Creating load file for flashing: .build/clueboard_66_rev2_default.hex                               [OK]
-Copying clueboard_66_rev2_default.hex to qmk_firmware folder                                        [OK]
-Checking file size of clueboard_66_rev2_default.hex                                                 [OK]
+Linking: .build/clueboard_66_rev3_default.elf                                                       [OK]
+Creating load file for flashing: .build/clueboard_66_rev3_default.hex                               [OK]
+Copying clueboard_66_rev3_default.hex to qmk_firmware folder                                        [OK]
+Checking file size of clueboard_66_rev3_default.hex                                                 [OK]
  * File size is fine - 25174/28672
```

Comment:
Should update this, as it's been updated in the code. 
```suggestion
 * The firmware size is fine - 26356/28672 (2316 bytes free)
```

Reply:
Good one, thanks!